content/docs/1.9.0/important-notes/_index.md

```diff
@@ -59,6 +61,21 @@ The `v1beta1` version of the Longhorn API is deprecated in v1.9.0 and will be re
 
 Deprecated APIs are no longer served and may therefore cause unexpected or unwanted behavior. Avoid using longhorn.io/v1beta1 in new code and, if possible, rewrite existing code to exclude this version.
 
+## Breaking Change
+
+### V2 Backing Image
+
+Due to naming conflicts in the extended attributes (xattrs) used by SPDK backing image logical volumes, the v1.9.0 V2 backing image is incompatible with previous versions.
+
+- **Before upgrading to v1.9.0**:
+  - Backup all volumes created using a V2 backing image.
+  - Delete these volumes after backup.
+- **After upgrading**:
+  - Recreate the necessary V2 backing images.
+  - Restore and rebuild volumes from your backups.
```

Comment:
@COLDTURNIP Please check if the workaround is valid before merging the PR. Thanks.